### PR TITLE
update readme and fix build warning

### PR DESCRIPTION
**Pull request by milestorme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,7 @@
 # CrossFaction Battleground
 
 ### Module currently requires:
-* **MODULE DON'T SUPPORT FOR AC OFFICIALLY NEED TEST THIS PR'S**
-* * https://github.com/azerothcore/azerothcore-wotlk/pull/2064
+* Need AC commit [`d40e8946`](https://github.com/azerothcore/azerothcore-wotlk/commit/d40e8946180129b39172c2a1b4d690aa71723917)
```
Comment (FrancescoBorzi) [inline README.md]: Maybe reword in something like "Requires this version of AC (or newer)"

Comment (milestorme) [reply README.md]: Done
